book/src/editor.md

```diff
@@ -31,6 +31,7 @@
 | `line-number` | Line number display: `absolute` simply shows each line's number, while `relative` shows the distance from the current line. When unfocused or in insert mode, `relative` will still show absolute line numbers | `absolute` |
 | `cursorline` | Highlight all lines with a cursor | `false` |
 | `cursorcolumn` | Highlight all columns with a cursor | `false` |
+| `continue-comment` | if helix should automatically add a (line-)comment token, if you create a new line inside a comment. | `false` |
```

Comment:
```suggestion
| `continue-comments` | if helix should automatically add a line comment token if you create a new line inside a comment. | `false` |
```

I'm not sure we can accurately continue block comments but if we do add that we can just edit the text here. I would also call this `continue-comments` instead and there's a stray `,` that can be removed here

Reply:
The option and comment in the code should be renamed to match this too

Reply:
Fair point. Changes applied.